app/models/menu_item.rb

```diff
@@ -0,0 +1,3 @@
+class MenuItem < ApplicationRecord
```

Comment:
validations for our models?

Reply:
Added validations and validation tests for our models!